src/components/Slug/Slug.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+
+const Container = styled.div`
+  background-color: #ffffff;
+  width: 100%;
+  padding: 30px 46px;
+`
+const InputContainer = styled.div`
```

Comment:
check these stylings to make sure there are none which you are settings which are already the default values